docs/swagger.yaml

```diff
@@ -769,6 +769,30 @@ paths:
         403:
           description: Forbidden.
   /repositories/{repo_name}/tags/{tag}:
+    get:
+      summary: Get the tag of the repository.
+      description: |
+        This endpoint aims to retrieve the tag of the repository.
```

Comment:
please add more description about details of tag. For example when Harbor is deployed with notary ....